### DISCUSSION TITLE
Online only meetings and plugin/page response

### DISCUSSION THREAD

**Discussion by troykru** [code4recovery/12-step-meeting-list#1110] [General]
Hi, new, working on transitioning from an old Base to the new 12 step plug in. 

01 - if a meeting doesn't have an address, it is showing as inactive. How do I mark it as online only and have it show up as active.
02 - the search via the drop down menus seems intermittent, and sluggish as best. - sometimes I will select a drop down and the page does not respond.... 

I appreciate any insight and help
Troy

**Comment by kiyote33** [comment]
Hi @troykru, note in the instructions on your meeting edit page how to configure the address info for an online meeting. Here's an example I pulled of a site I support: 
![image](https://user-images.githubusercontent.com/57125427/229310116-24bfb4ab-418f-4a9e-859c-7b977689ac15.png)
If you have caching set up for your website, that will greatly assist in resolving performance issues. I've used the W3 Total Cache plugin on a few sites with some success, but there are quite a few free ones in the WordPress directory to choose from. Your site hosting might also be able to provide some caching help.

Hope this helps...

**Reply by kiyote33** [reply]
Hi again @troykru, sorry it's been a busy Saturday!  I just saw your email. Yes, there's an easy way to show a meeting active and online only! The instructions I sent earlier assumed you had already entered the Online Details for the meeting. That's all you need to finish doing for your "Star - Sisters Talking About Recovery" meeting. Here's the piece setup for a meeting here in Alberta: 
![image](https://user-images.githubusercontent.com/57125427/229323008-b7441917-5e13-4285-9f7d-9dd913ed9879.png)

**Comment by troykru** [comment]
woof... =), erg, it's right in front of me.... and I'll look at the caching
as well, thank you and SO nice to know there is some support as I wade thru
this process.
As you can imagine, years of neglected data, case mismatchings (my pet
peace), duplicate entries, notes that make no sense...
But thanx be to God, I have the aptitude, just need some knowledge...
As you so graciously provided...
Can't thank you enough... I may be knocking again... =), but once we're up
and running, I'll fade away... =)
cheers.
--
Thank you,

Troy Krueger

**Comment by troykru** [comment]
Thanks so much for your time, and your response. I had found that out, and
then deleted my reply. I guess you didn’t see it deleted. In any event with
your help I have found what I need, thank you.

On Sat, Apr 1, 2023 at 7:44 PM Wayne Parker ***@***.***>
wrote:

> Hi again @troykru <https://github.com/troykru>, sorry it's been a busy
> Saturday! I just saw your email. Yes, there's an easy way to show a meeting
> active and online only! The instructions I sent earlier assumed you had
> already entered the Online Details for the meeting. That's all you need to
> finish doing for your "Star - Sisters Talking About Recovery" meeting.
> Here's the piece setup for a meeting here in Alberta:
> [image: image]
> <https://user-images.githubusercontent.com/57125427/229323008-b7441917-5e13-4285-9f7d-9dd913ed9879.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/1110#discussioncomment-5499303>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A646BEJTYOIGCLW5XITWALLW7C4WVANCNFSM6AAAAAAWPZVPJ4>
> .
> You are receiving this because you were mentioned.Message ID:
> <code4recovery/12-step-meeting-list/repo-discussions/1110/comments/5499303
> @github.com>
>
-- 
--
Thank you,

Troy Krueger

**Reply by troykru** [reply]
Thanx man, all good, here's an interesting find.
On my iPhone 14 premax, when I select "everywhere" from the dropdown, all the results are in a much larger font size than if I select a single region. screenshots attached, any idea why that is? The font size is that same on a computer. 

![AA Meetings in Campbell Hall](https://user-images.githubusercontent.com/129622161/229354517-d20146f6-7aa2-4c1d-88b5-d1e490c8955d.png)
![AA Meetings](https://user-images.githubusercontent.com/129622161/229354518-1a582acf-78c2-4f5f-9891-2d81d6c4c59b.png)

**Reply by joshreisner** [reply]
hi @troykru that's an unusual one! looks like your theme is applying some styles, try adding this CSS to override it:

```
#tsml .row {
  display: block;
  width: calc(100% + 30px);
}

#tsml .row td {
  white-space: normal !important;
  -webkit-text-size-adjust: 100%;
}
```

**Comment by troykru** [comment]
hi, thanx, I noticed I was not using the tsml settings, it was still the
legacy settings, when I changed over everything was fine and it looks a lot
better as well!
again, thank you
--
Thank you,

Troy Krueger
